src/simdutf/ppc64/simd.h

```diff
@@ -376,6 +376,15 @@ template <typename T> struct simd8x64 {
     this->chunks[3].store(ptr + sizeof(simd8<T>) * 3/sizeof(T));
   }
 
+
```

Comment:
The attribute chunks in the simd8x64 struct is marked as const. I think if you just remove the 'const' in 'const chunks', it should work?

Reply:
Yes, it works!

Reply:
Just not passing the tests yet, but it compiles.